docs/application/vstools/install.md

```diff
@@ -17,8 +17,8 @@ To work with Visual Studio Tools for Tizen, your computer must have:
 
 - At least 1.5 GB of available disk space
 - Visual Studio 2017 to use Tizen 4.0 and 5.0
-- Visual Studio 2019 to use Tizen 4.0 and higher
-- Visual Studio 2022 to use Tizen 6.5 and higher
+- Visual Studio 2019 to use Tizen 4.0 and 6.5
```

Comment:
```suggestion
- Visual Studio 2019 to use Tizen 4.0 and 6.5.
```

Reply:
@sangwook96 This comment was perhaps missed and not updated. Please look into this.

Reply:
To maintain consistency with the other bullet points, the sentence should end with a period. 
```suggestion
- Visual Studio 2019 to use Tizen 4.0 and 6.5.
```